common/metrics/client.go

```diff
@@ -129,3 +127,9 @@ func getMetricDefs(serviceIdx ServiceIdx) map[int]metricDefinition {
 
 	return defs
 }
+
+func mergeDictoRight(src map[string]string, dest map[string]string) {
```

Comment:
maybe name it mergeMapToRight? at least mergeDicToRight

Reply:
will do